src/main/java/com/cloudbees/jenkins/plugins/bitbucket/BitbucketBuildStatusNotifications.java

```diff
@@ -110,9 +114,11 @@ private static void createStatus(@NonNull Run<?, ?> build, @NonNull TaskListener
             return;
         }
 
-        String url;
+        String rootUrl, url;
         try {
-            url = getRootURL(build);
+            rootUrl = getRootURL();
```

Comment:
Root URL does not need a `checkURL`. There is no requirements for the text fields it is used in, and we want to keep whatever the user have put in there.